src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Generated/IOperationsOperations.cs

```diff
@@ -0,0 +1,47 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 0.16.0.0
```

Comment:
even after doing autorest --latest or autorest --reset 
the version that i get is 0.16.0.0.
How to i get 1.0?